BMM.Core/Utils/LoggingUtils.cs

```diff
@@ -0,0 +1,28 @@
+namespace BMM.Core.Utils;
+
+public static class LoggingUtils
+{
+    public static void AddParameter(this Dictionary<string, object> paramsDictionary,
```

Comment:
"This splits up a long string into multiple parameters."
Please add that (or similar) as comment